src/Marten.Testing/Harness/IntegrationContext.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using Marten.Testing.Examples;
+using Xunit;
+
+namespace Marten.Testing.Harness
+{
+    [CollectionDefinition("integration")]
+    public class IntegrationCollection : ICollectionFixture<DefaultStoreFixture>
+    {
+
+    }
+
+    [Collection("integration")]
+    public class IntegrationContext : StoreContext<DefaultStoreFixture>
```

Comment:
**CONCERN:** Why do we need few IntegrationContexts? Can't we reuse them? If they have different purposes then it would be good if they have different naming.

Reply:
There's a different integration context in the Schema project and yet another in the Noda, but they behave differently than the one, single context here

Reply:
Can we rename them to eg. HarnessIntegrationContext etc.?